packages/dds/map/src/test/memory/map.spec.ts

```diff
@@ -57,30 +33,49 @@ describe("SharedMap memory usage", () => {
         // See the comment at the top of the test suite for more details.
     });
 
-    benchmarkMemory({
-        title: "Create empty map",
-        minSampleCount: 1000,
-        benchmarkFn: async () => {
-            createLocalMap("testMap");
-        },
-    });
+    benchmarkMemory(new class implements IMemoryTestObject {
+        title = "Create empty map";
+        private map: SharedMap = createLocalMap("testMap");
+
+        async run() {
+            this.map = createLocalMap("testMap");
+        }
+    }());
 
     const numbersOfEntriesForTests = [1000, 10_000, 100_000];
 
     numbersOfEntriesForTests.forEach((x) => {
-        benchmarkMemory({
-            title: `Add ${x} integers to a local map`,
-            benchmarkFn: createTestForAddingIntegerEntries(x),
-        });
+        benchmarkMemory(new class implements IMemoryTestObject {
+            title = `Add ${x} integers to a local map`;
+            private map: SharedMap = createLocalMap("testMap");
+
+            async run() {
+                this.map = createLocalMap("testMap");
```

Comment:
By now you probably know what I'm going to say 😄 . Here, in the test below, and in the tests in Matrix and SharedString.